processor/resourcedetectionprocessor/internal/aws/eks/detector.go

```diff
@@ -132,12 +133,12 @@ func newK8sDetectorUtils() (*eksDetectorUtils, error) {
 	return &eksDetectorUtils{clientset: clientset}, nil
 }
 
-func (e eksDetectorUtils) getConfigMap(ctx context.Context, namespace string, name string) (map[string]string, error) {
-	cm, err := e.clientset.CoreV1().ConfigMaps(namespace).Get(ctx, name, metav1.GetOptions{})
+func (e eksDetectorUtils) getClusterVersion() (string, error) {
+	serverVersion, err := e.clientset.Discovery().ServerVersion()
```

Comment:
I see the [underlying api call](https://github.com/kubernetes/client-go/blob/v0.32.3/discovery/discovery_client.go#L658) is to `/version`. I wonder if this endpoint also needs explicit permission like below to be included in the clusterrole bound to a collector's service account?
```
- nonResourceURLs:
  - /version
  verbs:
  - get
  ```
  I think we should update the README if there is any specific rbac requirement.

Reply:
@jinja2 
I just looked it up and `/version` is part of the default permissions given to a service account.
Not sure if it's needed to explicitly define it?

Reply:
I think we should be good then. thanks for checking